Library/OcMainLib/OpenCoreMisc.c

```diff
@@ -470,8 +471,27 @@ OcMiscEarlyInit (
     gBS->SetWatchdogTimer (0, 0, 0, NULL);
   }
 
-  if (Config->Misc.Debug.SerialInit) {
+  if (Config->Misc.Serial.Init) {
     SerialPortInitialize ();
+
+    //
+    // Update PCD values.
```

Comment:
Shall we introduce any sanity checks before overriding these values? For example, BaudRate, PciDeviceInfo, etc...

Reply:
I don't think so. A value will work or not work. I suppose an insane value could cause a crash but I would put that in the "not work" category.

Reply:
Alright, I just moved such checks to ocvalidate. And we shall only check those explicitly mentioned in MdeModulePkg.dec.